Sources/Momento/internal/TopicsGrpcManager.swift

```diff
@@ -24,8 +24,10 @@ internal class TopicsGrpcManager {
                 // Note: Keepalive should in most circumstances not be necessary.
                 configuration.keepalive = ClientConnectionKeepalive(
                     interval: .seconds(10),
-                    timeout: .seconds(5)
+                    timeout: .seconds(5),
+                    permitWithoutCalls: true
                 )
+                configuration.connectionPool.connectionsPerEventLoop = 100
```

Comment:
this seemed to be the closest thing to the `maxConcurrentStreams` setting that we use in the go sdk, but not sure if it's needed here

Reply:
We'll probably want to do some testing on this at some point, but this seems fine for now as a callout at least.